README.md

```diff
@@ -17,6 +17,7 @@ You can choose how much Clippy is supposed to ~~annoy~~ help you by changing the
 | `clippy::style`       | code that should be written in a more idiomatic way                     | **warn**      |
 | `clippy::complexity`  | code that does something simple but in a complex way                    | **warn**      |
 | `clippy::perf`        | code that can be written to run faster                                  | **warn**      |
+| `clippy::suspicious`  | code that seems very likely to be incorrect                             | **warn**      |
```

Comment:
I would put this right under `correctness` and word it similar:

```text
| `clippy::suspicious`  | code that is very likely wrong or useless                        | **warn**      |
```

Reply:
Good idea. How about this? (removed a "very" and changed to "most likely")

```text
`clippy::correctness` | code that is outright wrong or useless
`clippy::suspicious`  | code that is most likely wrong or useless
```